src/pages/home/report/ReportActionCompose.js

```diff
@@ -206,8 +217,13 @@ class ReportActionCompose extends React.Component {
             suggestedEmojis: [],
             highlightedEmojiIndex: 0,
             colonIndex: -1,
-            shouldShowSuggestionMenu: false,
-            isEmojiPickerLarge: false,
+            shouldShowEmojiSuggestionMenu: true,
+            shouldShowMentionSuggestionMenu: true,
```

Comment:
Why do these default to `true`?

Reply:
I guess it doesn't matter in practice since `suggestedEmojis` and `suggestedMentions` are both `[]` but it still seems weird to set it to true.

Reply:
Good catch, I think these values are irrelevant, since they will be recalculated inside the `onSelectionChange` callback that fires on first load, but I'll change them back to false